packages/synapse-react-client/src/synapse-queries/entity/useEntity.ts

```diff
@@ -301,7 +304,7 @@ export function useUpdateViaJson(
       await invalidateAllQueriesForEntity(queryClient, keyFactory, entityId)
       queryClient.setQueryData(
         // This annotation data will never include derived annotations, which are calculated by the backend asynchronously
-        keyFactory.getEntityJsonQueryKey(entityId, false),
+        keyFactory.getEntityJsonQueryKey(entityId, undefined, false),
```

Comment:
Just to double check, but based on the updates to the query keys, this should invalidate all queries for the entity, including those with the version number?

Reply:
The invalidation will invalidate all queries for the entity. This `setQueryData` will only apply to the "versionless"/"current" annotations data